tests/integration/build_dir.rs

```diff
@@ -0,0 +1,14 @@
+use failure::Error;
```

Comment:
Hm? This file is not included anywhere, and `purge_on_drop` is not implemented.

Reply:
Oops, this slipped in from another branch. I removed it.